plugins/widget/plugin.js

```diff
@@ -2507,7 +2507,8 @@
 				if ( !target.type )
 					return false;
 
-				widget = widgetsRepo.getByElement( target );
+				widget = ( target instanceof CKEDITOR.dom.element ) ? widgetsRepo.getByElement( target ) : null;
```

Comment:
After adding this line, the above IE8 workaround seems to be not needed anymore. Could you check it and remove it if so?

The comment with a link should be moved here (please reformat it to the current convention). Please also link to #663 issue there.